### PR TITLE
Task to purge repo

### PR DESCRIPTION
**Pull request by cgalarza**

### CODE REVIEW

File: lib/tasks/bulwark/repo.rake [hunk 1]
```diff
@@ -0,0 +1,77 @@
+namespace :bulwark do
+  namespace :repo do
+    desc 'Purges Repo. Removes db object, git repository and special remotes.'
+    task purge: :environment do
+      abort(Rainbow('Incorrect arguments. Pass arklist=/path/to/file or arks=seperated,by,commas').red) if ENV['arklist'].blank? && ENV['arks'].blank?
+
+      # Pass in ARK(s) for digital object(s) to be deleted.
+      arks = open(ENV['arklist']).map(&:strip!) if ENV['arklist'].present?
```
Comment (mksndz) [inline lib/tasks/bulwark/repo.rake:8]: catching an exception here from `open()` (e.g., `FileNotFound`) would be helpful here if, say, the user mistypes a path

File: lib/tasks/bulwark/repo.rake [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+        puts Rainbow("Deleting #{ark}...").green
+
+        # Remove git repository.
+        remote_path = repo.version_control_agent.remote_path
+        begin
+          FileUtils.remove_dir(remote_path)
+          puts "  Git Repository: Removed"
+        rescue StandardError => e
+          puts Rainbow("  Git Repository: Error removing git repository from #{remote_path}").red
```
Comment (mksndz) [inline lib/tasks/bulwark/repo.rake:29]: why not also output `e.message` here?

File: lib/tasks/bulwark/repo.rake [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+            puts "  Ceph Bucket: Deleted"
+          rescue
+            puts Rainbow("  Ceph Bucket: Error deleting bucket").red
+          end
+        else
+          puts "  Ceph Bucket: Does not exist"
+        end
+
+        # Purposefully not deleting fedora object and solr documents because they will
+        # most likely be recreated.
```
Comment (mksndz) [inline lib/tasks/bulwark/repo.rake:53]: and leaving the repo in place won't cause issues, if, say metadata has changed in the repo?

Comment (cgalarza) [reply lib/tasks/bulwark/repo.rake:53]: The fedora object is deleted and recreated. Solr documents are the same way. There might be some cases where they will need to be deleted manually. But most of the time there won't be a Solr or Fedora object.

File: lib/tasks/bulwark/repo.rake (hunk 2 above)
Comment (cgalarza) [reply lib/tasks/bulwark/repo.rake:29]: I can add that in.

File: lib/tasks/bulwark/repo.rake (hunk 1 above)
Comment (cgalarza) [reply lib/tasks/bulwark/repo.rake:8]: Will do

**Review by mksndz — APPROVED** [review]
a few recommendations. but this looks like a helpful administration utility!

**Pull request merged**